src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,51 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+
 public class SalaryInfo {
+    private static final byte DATE_LENGTH = 3;
+    private static final byte YEAR_ARRAY_INDEX = 0;
+    private static final byte WORK_HOURS_ARRAY_INDEX = 2;
+    private static final byte SALARY_PERHOUR_ARRAY_INDEX = 3;
+    private static final byte EQUAL_YEAR_CONDITION = 0;
```

Comment:
I get that these values can fit in one byte and it works, but this is rather unusual to see byte constants in the context where there we don't deal with any bytes, and makes the reader wonder what was the reason behind it, I would recommend just go with `int`